packages/orbit-components/src/HorizontalScroll/README.md

```diff
@@ -33,6 +33,8 @@ After adding import into your project you can use it simply like:
 | onOverflow        | `() => void`                |          |                      | callback function, fires, if content is overflowed                            |
 | arrows            | `boolean`                   |          |                      | show arrows                                                                   |
 | arrowColor        | `string`                    |          |                      | set arrows color                                                              |
+| arrowLeftTitle    | `string`                    |          |                      | set title for left arrow                                                      |
+| arrowRightTitle   | `string`                    |          |                      | set title for right arrow                                                     |
```

Comment:
I would add mention of this to the docs, either as the Accessibility tab we have been adding to fixed components (e.g. [Seat](https://orbit.kiwi/components/visuals/seat/accessibility/), [Stepper](https://orbit.kiwi/components/interaction/stepper/accessibility/)), or in the react tab as a new section (e.g. in [Popover](https://orbit.kiwi/components/overlay/popover/react/#accessibility)).

Reply:
Added in a new commit, thanks for the great suggestion! ☺️